draft-ietf-httpbis-origin-frame.md

```diff
@@ -147,16 +159,16 @@ See {{algo}} for an illustrative algorithm for processing ORIGIN frames.
 The set of origins (as per {{!RFC6454}}) that a given connection might be used for is known in this
 specification as the Origin Set.
 
-By default, the Origin Set for a connection is uninitialised. When an ORIGIN frame is first received
-and successfully processed by a client, the connection's Origin Set is defined to contain an initial
-origin.  The initial origin is composed from:
+By default, the Origin Set for a connection is uninitialised. When the ORIGIN setting ({{setting}})
```

Comment:
I think that you want to change the "By default" here too.  Perhaps:

"On a connection where the server has not sent an ORIGIN_SUPPORT setting, the rules for authority in Section 9.1.2 of {{!RFC7540}} apply and the Origin Set is not used."  ... or something.

s/ORIGIN/ORIGIN_SUPPORT/

Reply:
I'll second that.  If the server doesn't declare support, return to the RFC the server knows it's implementing.